spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -2,7 +2,19 @@
 require 'rails_helper'
 
 RSpec.describe V0::SessionsController, type: :controller do
-  let(:saml_attrs) { { 'uuid' => ['1234'], 'email' => ['test@test.com'] } }
+  let(:saml_attrs) do
+    {
+      'uuid' => ['1234'],
+      'email' => ['test@test.com'],
+      'fname' => ['abraham'],
+      'lname' => ['lincoln'],
+      'mname' => [''],
+      'social' => ['111-22-3333'],
+      'birth_date' => ['1809-02-12']
+    }
+  end
+  # has an LOA of 'http://idmanagement.gov/ns/assurance/loa/2'
+  let(:response_xml) { File.read("#{::Rails.root}/spec/fixtures/files/saml_response.xml") }
```

Comment:
We needed a test to cover the LOA parsing code (`sessions_controller.level_of_assurance`).  To do so, I created a minimalist version of the SAML Response XML.  I'm not sure if this is the cleanest way to do this and I'm open to any suggestions.